app/entities/account.go

```diff
@@ -0,0 +1,119 @@
+package entities
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Represents a user account in the digital bank.
+type Account struct {
+	id         string
+	name       string
+	cpf        string
+	secret     string
+	balance    int64
+	created_at time.Time
+}
+
+// constructor
+func NewAccount(id, cpf, name, secret string) (*Account, error) {
+
+	//Creates a new instance of Account.
+	account := &Account{
+		id:         id,
+		cpf:        cpf,
+		name:       name,
+		secret:     secret,
+		balance:    0,
+		created_at: time.Now(),
+	}
+
+	return account, nil
+}
+
+// Check if any fields are empty.
+func EmptyField(cpf, name, secret string) bool {
+	if cpf == "" || name == "" || secret == "" {
+		return true
+	}
+	return false
+}
+
+// Validates a Brazilian CPF.
+func ValidaCPF(cpf string) bool {
```

Comment:
Como essa função valida algo, acho mais interessante retornar um erro em vez de booleano. Caso o erro seja `nil`, quer dizer q o cpf é válido e, caso contrário, inválido. Retornar um erro ajuda a entender oq há de errado com o cpf
```suggestion
func ValidaCPF(cpf string) error {
```